src/main/java/uk/gov/hmcts/sptribs/ciccase/model/CicCase.java

```diff
@@ -369,6 +369,12 @@ public class CicCase {
     private CaseDocumentsCIC reinstateDocuments;
     private YesOrNo selectedCheckBox;
 
+    @CCD(
+        label = "",
+        access = {DefaultAccess.class, CaseworkerWithCAAAccess.class}
+    )
+    private CaseDocumentsCIC orderFile;
+
```

Comment:
Do we need this in CicCase.java file. Think we are using orderFile attribute defined in SendOrder.java class.